SpectatorView/Compositor/CompositorDLL/CompositorInterface.cpp

```diff
@@ -172,7 +173,7 @@ void CompositorInterface::TakePicture(ID3D11Texture2D* outputTexture)
 
     // Get bytes from texture because screengrab does not support texture format provided by Unity.
     DirectXHelper::GetBytesFromTexture(_device, outputTexture, FRAME_BPP, photoBytes);
-    ID3D11Texture2D* tex = DirectXHelper::CreateTexture(_device, photoBytes, FRAME_WIDTH, FRAME_HEIGHT, FRAME_BPP, DXGI_FORMAT_B8G8R8A8_UNORM_SRGB);
+    ID3D11Texture2D* tex = DirectXHelper::CreateTexture(_device, photoBytes, FrameProviderStaticConfig::width, FrameProviderStaticConfig::height, FRAME_BPP, DXGI_FORMAT_B8G8R8A8_UNORM_SRGB);
```

Comment:
How do these dimensions fall back to FRAME_WIDTH/ FRAME_HEIGHT if the config file is not present?
Looks like width and height will be undefined.  

Can you make width and height private, then the function get_width() and get_height() should return the cached loaded value (if file existed) or the fallback constant if not.

Reply:
variables are initialized to FRAME_WIDTH/FRAME_HEIGHT inside CompositorDLL. They are `__declspec(dllexport/importd)`ed from that DLL. So if they aren't touched by the code that load the config file, they are set to whatever was set in the header.

Also, the configuration file loader class throws exceptions when it cannot read the file. I should explicitly reset theses values to FRAME_WIDTH/FRAME_HEIGHT/VIDEO_FPS in the catch() statements

Reply:
Some of the source files have been encoded in Unicode (2byte per character on windows). Git thinks they are binary so GitHub doesn't really show `diff` correctly on theses files. 

Compositor.DLL contains the static initialization of theses files 

```cpp

#include "FrameProviderStaticConfig.h"

int FrameProviderStaticConfig::width = FRAME_WIDTH;
int FrameProviderStaticConfig::height = FRAME_HEIGHT;
float FrameProviderStaticConfig::fps = VIDEO_FPS;
```

Reply:
In either case, now this is specifically handled when catching an exception if there's any problem with reading the config file